cmd/loom/loom.go

```
@@ -730,6 +733,17 @@ func loadApp(
 		return nil, err
 	}
 
+	if build := appStore.Get(minimumBuildKey); !bytes.Equal(build, []byte{}) {
+		minimumBuild := binary.BigEndian.Uint64(build)
+		currentBuild, err := strconv.ParseUint(loomchain.Build, 10, 64)
+		if err != nil {
```

I think we should return the error in this case, wrap it first though
```golang
return nil, errors.Wrap(err, "failed to parse loomchain build number")
```